src/design-guidelines/disabled-hidden/examples/disabled-vs-hidden.tsx

```diff
@@ -127,7 +127,7 @@ export class ButtonDisabledVsHideExample {
                 button, it is better to show the <b> unassign</b> button
                 instead.
             </p>,
-            <div class="relevan-buttons-example">
+            <div class="relevant-buttons-example">
```

Comment:
<img width="951" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/4524c66f-4816-49e0-bf45-53d56a1b9bac">

If you use the same classes and HTML structure that we use in the other examples ☝️ , then you don't need to make special CSS for this example. It's more maintainable to have the same source for all of these layouts, and not treat them individually. 😊

Reply:
as you see, they are wrapped in a div with class of `do-dont-container` which has all the necessary styles. 

Also make sure to remove any replicated styles which we don't need.

Reply:
The generic (from other examples) one didn't help me here, then would it be better just to have a customer here?

Reply:
I mean, I use some CSS from that import. But not all work for my examples like ` background-color: rgb(var(--contrast-100));` I have overwritten it here. 
I tried to use only `import` but it didn't bring desirable results.

Reply:
made a fixup

Reply:
this is great now. 

It's alright to overwrite things if needed. My point was not to force styles that aren't needed. ;) 

There is just this problem now, but it has nothing to do with your code. It's just a poor CSS that I wrote back in the day. 😅 👇 

<img width="534" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/a6c2e8e0-6168-460b-b64d-9fcf769ff04f">

we will fix it later